src/scanoss/cli.py

```diff
@@ -315,6 +315,8 @@ def setup_args() -> None:
 
     # Inspect Sub-command: inspect undeclared
     p_undeclared = p_inspect_sub.add_parser('undeclared', aliases=['un'],description="Inspect for undeclared components", help='Inspect for undeclared components')
+    p_undeclared.add_argument('--style',required=False ,choices=['sbom', 'scanoss-settings'],
```

Comment:
I'm not sure if `style` is the best name here, probably i would use something like `settings-file-format` and 'legacy' (sbom.json) or 'new' (scanoss.json) for the options to apply the same convention already used in `scanoss_settings.py`

(i'm not sure this is the best naming neither)

Reply:
should we use '—settings-type' ? @matiasdaloia ?